### DISCUSSION TITLE
Windows: Unable to read config files for spawned subprocess when run as LocalSystem

### DISCUSSION THREAD

**Discussion by T-256** [rustdesk/rustdesk#9290] [General]
It seems configs not respected when system-privileged Rustdesk process spawns subprocess as regular user.
https://github.com/rustdesk/rustdesk/blob/c8cd564e6986b1aab9ed066f678984b1e852e96b/src/server/connection.rs#L1684-L1688

For Reproduce:
1. enable `hide-tray` option
1. run `rustdesk --server` as interactive LocalSystem or Service
1. it correctly won't show tray icon for now
1. connect from another machine
1. 🐛 now tray icon appears on user taskbar

**Comment by rksdutt** [comment]
Verify the config file path like that : icacls "C:\path\to\configfile" /grant "SYSTEM:R" or dir C:\path\to\directory
And give it a permission : icacls "C:\path\to\config\folder" /grant "SYSTEM:(OI)(CI)F"

**Reply by T-256** [reply]
When run as `NT AUTHORTIY\SYSTEM`, the default config path for rustdesk config folder is `C:\Windows\ServiceProfiles\LocalService\AppData\Roaming\RustDesk\config`.
When I connect to it, it won't be able to read from above path, then uses `C:\Users\User\Appdata\Roaming\RustDesk\config` (which has different settings).
